examples/squeezenet/SqueezeNet.js

```diff
@@ -145,26 +163,38 @@ class SqueezeNet {
         }
         data = nhwcData;
       }
+
+      // shape tensor for reshape
+      if (initializer.dims.length === 1 && initializer.int64Data.length) {
+        data = new Int32Array(initializer.int64Data);
+      }
```

Comment:
would we show some warnings for other tensor types or shapes?

Reply:
I will add some warnings to type casting like this.